NetworkService/Sources/NetworkService/NetworkService.swift

```diff
@@ -0,0 +1,8 @@
+final class NetworkService {
+    /// Stub method to check possibility to use this NetworkService in your App.
+    ///
+    ///  - Returns: true
+    func use() -> Bool {
```

Comment:
nit:

Why do we need the return value?
```suggestion
    func use() {
```

Reply:
Probably we don't need to return anything.
We will use this method only to check its availability after the package is connected.

Reply:
deleted return value